server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -316,6 +314,12 @@ public class Node implements Closeable {
         }
     }, Setting.Property.NodeScope);
 
+    public static final Setting<ByteSizeValue> NODE_SEARCH_CACHE_SIZE_SETTING = Setting.byteSizeSetting(
```

Comment:
Is this gated by the feature flag? If not then it probably needs to be.

Reply:
It's not currently. Let me add that in.